modules/client-configuration/pages/supported-features-sle-micro.adoc

```diff
@@ -8,6 +8,10 @@ The operating system you run on a client is supported by the organization that s
 {micro} is supported by {suse}.
 ====
 
+[IMPORTANT]
+====
+{micro} is only supported as regular minion for the time being; we are working on managing it as Salt SSH client.
```

Comment:
```suggestion
{micro} is only supported as regular minion ([systemitem]``default`` contact method) for the time being. We are working on managing it as Salt SSH client ([systemitem]``salt-ssh`` contact method), too.
```
In the documentation, we do not use the term "regular minion".  For clarity, I propose to add the literal settings.